packages/coil-oauth-scripts/package.json

```diff
@@ -33,6 +33,7 @@
     "serve-it": "ts-node -P ../../tsconfig.json -r tsconfig-paths/register -T src/bin/serve-it.ts",
     "test": "jest --passWithNoTests",
     "test:coverage": "yarn test --verbose --coverage",
+    "test:e2e": "yarn test --config jest-e2e.config.js",
```

Comment:
would need to run this in the CI at some step

Reply:
Old comment :)  I just chucked this up PR as an example for you, but then decided to try and 'finish' it yesterday

Will see about getting `src/**/*.test.ts` for the unit tests sorted out